### PR TITLE
Add CI and Software Quality

### PR DESCRIPTION
**Pull request by ligurio**
**[https://github.com/ligurio/awesome-ci]**
**[https://github.com/ligurio/awesome-software-quality]**

It's an awesome list of Continuous Integration services and awesome list of free software testing and verification resources.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -621,6 +622,7 @@
 - [JMeter](https://github.com/aliesbelik/awesome-jmeter#readme) - Load testing and performance measurement tool.
 - [Creative Coding](https://github.com/terkelg/awesome-creative-coding#readme) - Programming something expressive instead of something functional.
 - [No-Login Web Apps](https://github.com/aviaryan/awesome-no-login-web-apps#readme) - Web apps that work without login.
+- [Software Quality (Testing, Verification)](https://github.com/ligurio/awesome-software-quality)
 - [Testing](https://github.com/TheJambo/awesome-testing#readme) - Software testing.
```
Comment (pajaydev) [inline readme.md]: Kindly add a description for this, something like this

```[Software Quality (Testing, Verification)](https://github.com/ligurio/awesome-software-quality) - List of free software testing and verification resources```

**Comment by maehr** [comment]
@ligurio You should add instructions to work on your list in the form of a `contributing.md` file.

**Comment by caufieldjh** [comment]
https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md

**Comment by ligurio** [comment]
> @ligurio You should add instructions to work on your list in the form of a contributing.md file.

@maehr, done

**Comment by maehr** [comment]
> > @ligurio You should add instructions to work on your list in the form of a contributing.md file.
> 
> @maehr, done

@ligurio you should add a `contributing.md` that reflects the repos content (for example there are no tests present). Do something like https://github.com/maehr/awesome-digital-history/blob/master/CONTRIBUTING.md

**Comment by sindresorhus** [comment]
One PR per list addition please.

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Pull request closed without merging**